src/history/history.service.ts

```diff
@@ -93,6 +92,16 @@ export class HistoryService {
     return;
   }
 
+  async deleteHistory(user: User): Promise<void> {
+    const entries: HistoryEntry[] = await this.getEntriesByUser(user);
+    if (!entries) {
```

Comment:
When does `getEntriesByUser` return a nullish value?

Reply:
Never, it's just a habit because `findOne` returns `undefined`.